### ISSUE TITLE
Error on IDE startup (IntelliJ IDEA 2021.3)

### ISSUE THREAD

**Issue by vivekmore** [xylo/intellij-postfix-templates#206]
Error 1:
```
com.intellij.diagnostic.PluginException: Cannot create extension (class=de.endrullis.idea.postfixtemplates.language.CptLanguageCodeStyleSettingsProvider) [Plugin: de.endrullis.idea.postfixtemplates]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:931)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:88)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapter(ExtensionPointImpl.java:489)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:437)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.computeExtensionList(ExtensionPointImpl.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:243)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:39)
	at com.intellij.psi.codeStyle.CodeStyleSettingsServiceImpl.getLanguageCodeStyleProviders(CodeStyleSettingsServiceImpl.java:70)
	at com.intellij.psi.codeStyle.CommonCodeStyleSettingsManager.initNonReadSettings(CommonCodeStyleSettingsManager.java:105)
	at com.intellij.psi.codeStyle.CommonCodeStyleSettingsManager.getCommonSettingsMap(CommonCodeStyleSettingsManager.java:74)
	at com.intellij.psi.codeStyle.CommonCodeStyleSettingsManager.addLanguageSettings(CommonCodeStyleSettingsManager.java:275)
	at com.intellij.psi.codeStyle.CodeStyleSettings.registerCommonSettings(CodeStyleSettings.java:1213)
	at com.intellij.sql.dialects.mssql.formatter.MsVsSqlCodeStyleScheme.<clinit>(MsVsSqlCodeStyleScheme.kt:21)
	at com.intellij.sql.dialects.mssql.MsCodeStyleProvider.<init>(MsCodeStyleProvider.kt:14)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:830)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.instantiateClass(XmlExtensionAdapter.java:99)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter$SimpleConstructorInjectionAdapter.instantiateClass(XmlExtensionAdapter.java:116)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:66)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapter(ExtensionPointImpl.java:489)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:437)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.computeExtensionList(ExtensionPointImpl.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:243)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:39)
	at com.intellij.sql.formatter.SqlCodeStyleProviderRegistrar.<init>(SqlCodeStyleProviderRegistrar.kt:9)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:47)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:876)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:47)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:37)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:113)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:67)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:60)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateService(ComponentManagerImpl.kt:1044)
	at com.intellij.serviceContainer.ComponentManagerImpl$preloadServices$1.invoke(ComponentManagerImpl.kt:1016)
	at com.intellij.serviceContainer.ComponentManagerImpl$preloadServices$1.run(ComponentManagerImpl.kt:56)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: com.intellij.diagnostic.PluginException: Cannot create class de.endrullis.idea.postfixtemplates.language.CptLanguageCodeStyleSettingsProvider (classloader=PluginClassLoader(plugin=PluginDescriptor(name=Custom Postfix Templates, id=de.endrullis.idea.postfixtemplates, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-1/213.5744.223/IntelliJ IDEA.app.plugins/intellij-postfix-templates, version=2.12.1.213, package=null, isBundled=false), packagePrefix=null, instanceId=319, state=active))
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:869)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.instantiateClass(XmlExtensionAdapter.java:99)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter$SimpleConstructorInjectionAdapter.instantiateClass(XmlExtensionAdapter.java:116)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:66)
	... 42 more
Caused by: java.lang.IllegalAccessException: no such constructor: de.endrullis.idea.postfixtemplates.language.CptLanguageCodeStyleSettingsProvider.<init>()void/newInvokeSpecial
	at java.base/java.lang.invoke.MemberName.makeAccessException(MemberName.java:959)
	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1101)
	at java.base/java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:2030)
	at java.base/java.lang.invoke.MethodHandles$Lookup.findConstructor(MethodHandles.java:1264)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:830)
	... 45 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/application/options/IndentOptionsEditorBase
	at java.base/java.lang.invoke.MethodHandleNatives.resolve(Native Method)
	at java.base/java.lang.invoke.MemberName$Factory.resolve(MemberName.java:1070)
	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1098)
	... 48 more
Caused by: java.lang.ClassNotFoundException: com.intellij.application.options.IndentOptionsEditorBase PluginClassLoader(plugin=PluginDescriptor(name=Custom Postfix Templates, id=de.endrullis.idea.postfixtemplates, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-1/213.5744.223/IntelliJ IDEA.app.plugins/intellij-postfix-templates, version=2.12.1.213, package=null, isBundled=false), packagePrefix=null, instanceId=319, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:235)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 51 more
```

Error 2:
```
com.intellij.diagnostic.PluginException: Cannot create extension (class=de.endrullis.idea.postfixtemplates.language.CptLanguageCodeStyleSettingsProvider) [Plugin: de.endrullis.idea.postfixtemplates]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:931)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:88)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapter(ExtensionPointImpl.java:489)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:437)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.computeExtensionList(ExtensionPointImpl.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:243)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:39)
	at com.intellij.psi.codeStyle.LanguageCodeStyleSettingsProvider.calcSettingPagesProviders(LanguageCodeStyleSettingsProvider.java:385)
	at com.intellij.psi.codeStyle.LanguageCodeStyleSettingsProvider.lambda$getSettingsPagesProviders$2(LanguageCodeStyleSettingsProvider.java:379)
	at java.base/java.util.concurrent.atomic.AtomicReference.updateAndGet(AtomicReference.java:209)
	at com.intellij.psi.codeStyle.LanguageCodeStyleSettingsProvider.getSettingsPagesProviders(LanguageCodeStyleSettingsProvider.java:379)
	at com.intellij.psi.codeStyle.CodeStyleSettingsServiceImpl.getCustomCodeStyleSettingsFactories(CodeStyleSettingsServiceImpl.java:64)
	at com.intellij.psi.codeStyle.CustomCodeStyleSettingsManager.initCustomSettings(CustomCodeStyleSettingsManager.java:25)
	at com.intellij.psi.codeStyle.CodeStyleSettings.<init>(CodeStyleSettings.java:101)
	at com.intellij.psi.codeStyle.CodeStyleSettingsManager.createSettings(CodeStyleSettingsManager.java:52)
	at com.intellij.psi.codeStyle.ProjectCodeStyleSettingsManager.setMainProjectCodeStyle(ProjectCodeStyleSettingsManager.java:79)
	at com.intellij.psi.codeStyle.ProjectCodeStyleSettingsManager.<init>(ProjectCodeStyleSettingsManager.java:37)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:52)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:876)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:47)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:37)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:113)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:67)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:60)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateService(ComponentManagerImpl.kt:1044)
	at com.intellij.serviceContainer.ComponentManagerImpl$preloadServices$1.invoke(ComponentManagerImpl.kt:1016)
	at com.intellij.serviceContainer.ComponentManagerImpl$preloadServices$1.run(ComponentManagerImpl.kt:56)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: com.intellij.diagnostic.PluginException: Cannot create class de.endrullis.idea.postfixtemplates.language.CptLanguageCodeStyleSettingsProvider (classloader=PluginClassLoader(plugin=PluginDescriptor(name=Custom Postfix Templates, id=de.endrullis.idea.postfixtemplates, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-1/213.5744.223/IntelliJ IDEA.app.plugins/intellij-postfix-templates, version=2.12.1.213, package=null, isBundled=false), packagePrefix=null, instanceId=319, state=active))
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:869)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.instantiateClass(XmlExtensionAdapter.java:99)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter$SimpleConstructorInjectionAdapter.instantiateClass(XmlExtensionAdapter.java:116)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:66)
	... 35 more
Caused by: java.lang.IllegalAccessException: no such constructor: de.endrullis.idea.postfixtemplates.language.CptLanguageCodeStyleSettingsProvider.<init>()void/newInvokeSpecial
	at java.base/java.lang.invoke.MemberName.makeAccessException(MemberName.java:959)
	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1101)
	at java.base/java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:2030)
	at java.base/java.lang.invoke.MethodHandles$Lookup.findConstructor(MethodHandles.java:1264)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:830)
	... 38 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/application/options/IndentOptionsEditorBase
	at java.base/java.lang.invoke.MethodHandleNatives.resolve(Native Method)
	at java.base/java.lang.invoke.MemberName$Factory.resolve(MemberName.java:1070)
	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1098)
	... 41 more
Caused by: java.lang.ClassNotFoundException: com.intellij.application.options.IndentOptionsEditorBase PluginClassLoader(plugin=PluginDescriptor(name=Custom Postfix Templates, id=de.endrullis.idea.postfixtemplates, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-1/213.5744.223/IntelliJ IDEA.app.plugins/intellij-postfix-templates, version=2.12.1.213, package=null, isBundled=false), packagePrefix=null, instanceId=319, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:235)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 44 more
```

![image](https://user-images.githubusercontent.com/2099329/144169320-1cb70af1-fc96-4b3f-8a27-5fd09f483187.png)

IntelliJ IDEA 2021.3 (Ultimate Edition)
Build #IU-213.5744.223, built on November 27, 2021
Runtime version: 11.0.13+7-b1751.19 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.6.1
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 12
Kotlin: 213-1.5.10-release-949-IJ5744.223

**Comment by VarunS2002** [comment]
@vivekmore 
Same issue here

**Comment by xylo** [comment]
Fixed in version 2.12.2.
